app/scripts/controllers/landing.js

```diff
@@ -44,6 +45,11 @@ angular.module('confRegistrationWebApp')
         };
         $scope.searchComplete = val;
         $scope.searchResults = response.data;
+
+        analytics.digitalData.searchFilter = '';
```

Comment:
Currently the backend isn't sharing the SSO Guid, GR ID, or Facebook ID so those are placeholders for now. I do have access to the login provider (`RELAY` or `FACEBOOK`). Where would you like me to put that in digitalData?

I'll fix the only tracking on logged issue and am looking into adding the other filters.